crates/devolutions-pedm-shell-ext/AppxManifest.xml

```diff
@@ -10,10 +10,10 @@
     xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
     xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
     IgnorableNamespaces="uap uap2 uap3 rescap desktop desktop4 desktop5 uap10 com">
-    <Identity Name="DevolutionsAgent" ProcessorArchitecture="neutral" Publisher="CN=Devolutions"
-        Version="1.0.0.0" />
+    <Identity Name="DevolutionPEDMShellExtension" ProcessorArchitecture="neutral" Publisher="CN=Devolutions"
+        Version="2024.3.6.0" />
     <Properties>
-        <DisplayName>Devolutions Agent Shell Extension</DisplayName>
+        <DisplayName>Devolutions Agent</DisplayName>
```

Comment:
should we call it Devolutions Agent without "shell extension" since it's not actually Devolutions Agent, but the shell extensions for Devolutions Agent?

Reply:
Yes and no :) 

I'm not sure how the display name actually matters for the .msix since it's not something the user typically interacts with and I'm not sure where Windows actually shows you this (install MSIX / shell extensions)

However; if the shell extension launches the desktop agent (the tray icon application) and causes it to fire a UWP notification, the notification gets the name of the parent process. So you get a notification titled "Devolutions Agent Shell Extension" which is a bit of a weird experience.

I'm happy to change this back if there's a concern, though.